src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerCreateDraftOrder.java

```diff
@@ -164,8 +165,9 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(
         CaseDetails<CaseData, State> detailsBefore
     ) {
 
-        var caseData = orderService.generateOrderFile(details.getData(), details.getId());
-
+        Calendar cal = Calendar.getInstance();
+        String date = simpleDateFormat.format(cal.getTime());
+        var caseData = orderService.generateOrderFile(details.getData(), details.getId(), date);
```

Comment:
Can we not use Calendar? It's really well out of date. Use LocalDate.

Reply:
Changed from Calendar to LocalDateTime.